manifests/gem.pp

```diff
@@ -30,7 +30,7 @@
 #   Skips the installation of ri and rdoc docs.
 #   Default: false
 #   This variable is optional.
-
+#
```

Comment:
extra #

Reply:
This was accidentally included.  I don't mind if it is removed.